front/pages/w/[wId]/u/gens/index.tsx

```diff
@@ -1625,17 +1548,126 @@ export default function AppGens({
 
                 <div className="mb-4 mt-2 flex flex-row flex-wrap items-center text-xs font-normal"></div>
               </div>
+              <div className="mt-24 w-1/3">
```

Comment:
The Retrieval should be aligned with the textarea not below it or even aligned at the top?